### ISSUE TITLE
[classic] lua error when enabled the search in tooltip checkbox

### ISSUE THREAD

**Issue by doneykoo** [doneykoo/xmerchant-wow-addon#8]
**Describe the bug**
In classic version, lua error when enabled the search in tooltip checkbox.

**To Reproduce**
1. Go to any merchant
2. Talk with NPC
3. enable the search in tooltip checkbox
4. See error:
```
Interface\AddOns\xMerchant\core.lua:821: Usage: PlaySound(soundKitID, optional["SFX","Music","Ambience", "Dialog" or "Master"], [forceNoDuplicates, runFinishCallback])
```

**Expected behavior**
No error

**Screenshots**
n/a

**Version:**
 - wow 1.14.3
 - Server locale region US
 - OS Win

**Comment by doneykoo** [comment]
released 1.10.2-release-wotlk for WotLK, so it's now fixed in `Patch-classic-wrath`

**Comment by doneykoo** [comment]
Released 1.10.3

* Fixed compatible issues for all patches.
